.github/workflows/generate-import.yml

```diff
@@ -0,0 +1,43 @@
+name: Import File Generator
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "scripts/**/*"
+      - "styles/**/catppuccin.user.css"
+
+env:
+  HAS_USERSTYLES_TOKEN: ${{ secrets.USERSTYLES_TOKEN != '' }}
+
+jobs:
+  generate-health-files:
+    name: Generate health files
```

Comment:
probably should have a different job name here, but won't matter if we refactor the workflows.